Alignment/MuonAlignment/test/make_SVGtemplates_cfg.py

```diff
@@ -4,7 +4,7 @@
 process.source = cms.Source("EmptySource")
 process.maxEvents = cms.untracked.PSet(input = cms.untracked.int32(1))
 
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
```

Comment:
If I am not mistaken a Global Tag (or at least an ESSource providing the geometry records in the ES) is needed in order for `GeometryDB_cff` to work. 
As far as I can see, that is not included in this file (and in many others of this PR).